src/main/java/duke/datetime/Date.java

```diff
@@ -0,0 +1,59 @@
+package duke.datetime;
+
+/**
+ * Represents a date in the format Day/Month/Year.
+ */
+public class Date {
```

Comment:
I'm not sure if you have already considered this, but would it be better to use the built-in Java Date class?